portfolio/src/main/webapp/index.html

```diff
@@ -90,7 +90,7 @@ <h1 align="center">Ariel Salazar</h1>
           </tr>
       </table>
       <br>
-      <button onclick="getData()"> Fetch Data </button>
       <div id="quote-container"></div>
+      <p id="demo" onclick="getValues()"> Fetch The Values </p>
```

Comment:
Out of curiosity, why use a <p> tag here? It seems strange, since it's not a paragraph of text.